operations/local-replication/README.md

```diff
@@ -0,0 +1,499 @@
+
+
+# check performance secondary for access to secrets etc
+vault2 auth -method=userpass username=drtest password=drtest
+vault2 read supersecret/drtest
+
+# note that the .vault-token has changed
+diff ~/.vault-token ~/.vault-token-DRTEST
+
+## Promote DR secondary to primary
+vault3 write -f /sys/replication/dr/secondary/promote dr_operation_token=${DR_OPERATION_TOKEN}
+
+# check status
+vault3 read -format=json sys/replication/status
+
+# let's check our token status
+cp ~/.vault-token-DRTEST ~/.vault-token
+vault3 read supersecret/drtest
+
+# vault3 read supersecret/drtest
+## Key             	Value
+## ---             	-----
+## refresh_interval	768h0m0s
+## password        	baines
+## username        	harold
+
+## SUCCESS!
+
+```
+
+The environment looks like the following at this step:
+
+```
++---------------------------------+                    +------------------------------------+
+| vault port:8200                 |                    | vault2 port:8202                   |
+| Replication disabled            |                    | Performance secondary replication  |
+| (or demoted)                    |                    | vault3 --> vault2                  |
+|                                 |                    |                                    |
++---------------------------------+                    +------------------------------------+
+
+                                                                          ^
+                                                                          |
+                                                                          |
+                                                                          |
++---------------------------------+                                       |
+| vault3 port:8204                |                                       |
+| DR primary replication          |  +------------------------------------+
+| Performance primary replication |
+| vault3 --> vault2               |
++---------------------------------+
+```
+
+
+FAILBACK - Option 1  - relevant for Vault 0.8-0.9
+Note that this is not an idea situation today, as we must first sync DR replication set back to vault, then perform another failover such that vault is the perf primary/dr primary.
+```
+# Disable replication on vault (if not already done)
+vault write -f /sys/replication/dr/primary/disable
+vault write -f /sys/replication/performance/primary/disable
+
+# enable vault as DR secondary to vault3
+vault3 auth root
+vault3 write -f /sys/replication/dr/primary/enable
+PRIMARY_DR_TOKEN=$(vault3 write -format=json /sys/replication/dr/primary/secondary-token id=vault | jq --raw-output '.wrap_info .token' )
+sleep 10
+vault auth root
+vault write /sys/replication/dr/secondary/enable token=${PRIMARY_DR_TOKEN}
+sleep 10  
+```
+
+FAILBACK - Option 2 - relevant for Vault >= 0.9.1
+This scenario assumes the primary was demoted
+```
+# Promote original Vault instance back to disaster recovery primary
```

Comment:
I think a step is missed here. We never made the original instance a DR secondary of vault3. This is needed to make sure it gets any updates that went to vault3 while it was primary.